steam.profile: fix breakage with newer Proton-GE (process_vm_readv)

kmk3 opened the pull request:
As reported by @rsramkis on #5185, upgrading from Proton-7.2-GE-2[1]
(released on 2022-02-14) to GE-Proton7-18[2] (released on 2022-05-19)
breaks logging in on World of Tanks Blitz unless the `process_vm_ready`
32-bit syscall is allowed[3], so allow it.

Fixes #5185.

[1] https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/7.2-GE-2
[2] https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton7-18
[3] https://github.com/netblue30/firejail/issues/5185#issuecomment-1152350336

kmk3 commented:
@rsramkis @X6B

Can you confirm that this change is enough to fix the issues?

rsramkis commented:
> @rsramkis @X6B
> 
> Can you confirm that this change is enough to fix the issues?

Hi kmk3

I can confirm I am using the following lines in my steam.local file to launch World of Tanks Blitz when using Steam and Proton

```
#Used for WOT Blits with Proton > 7.2
seccomp.32 !process_vm_readv

```

kmk3 commented:
@rsramkis commented [on Aug 20](https://github.com/netblue30/firejail/pull/5323#issuecomment-1221368394):

> > @rsramkis @X6B Can you confirm that this change is enough to fix the
> > issues?
> 
> Hi kmk3
> 
> I can confirm I am using the following lines in my steam.local file to launch
> World of Tanks Blitz when using Steam and Proton
> 
> ```
> #Used for WOT Blits with Proton > 7.2
> seccomp.32 !process_vm_readv
> ```

Thanks for the confirmation.

There was no confirmation that this fixes the other games, so I removed those
references from the commit message.

Also, the previous Proton-GE versioning scheme was confusing (I wasn't sure if
you really meant 7.2 or just 7.0-2) and the versioning change added to the
confusion, so I added a bit more detail to the commit message.

Naming scheme change explanation for reference:

From <https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton7-1>:

> Why the new naming scheme?
> 
> As discussed here:
> 
> https://www.patreon.com/posts/63101415
> 
> The base for WINE was changed from wine upstream to Proton experimental, and
> staging currently is not used in this new build. There are several reasons
> this was done, which are explained in detail in the post above.
> 
> I needed a way to express that the base is all Proton7 and to reset the minor
> versions. Additionally, from previous discussions I've had with some other
> devs on official proton, they requested that I modify the name so that the GE
> versions were easier to differentiate, which is why the GE prefix has been
> moved to the front of the name.

Misc: The newer scheme seems easier to understand, especially with Proton-GE
now being based off of Proton experimental rather than the latest wine version
(which would usually have a higher/different version number compared to the
latest stable Proton release).
